fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/TransactionImpl.java

```diff
@@ -88,6 +89,10 @@ public synchronized void commit() {
             this.getEventAccumulator().emitEvents(id, baseUri, userAgent);
             this.committed = true;
         } catch (final PersistentStorageException ex) {
+            if (log.isDebugEnabled()) {
```

Comment:
A few questions here:

1. Why are you custom formatting the log message rather than just doing `log.debug("Failed to commit transaction {}", id, ex);`?
2. Why debug? I would consider this an error. Unless it's innocuous and happens all the time, but, if that's the case, there's a different set of questions.